### ISSUE TITLE
“Meaning” field no longer auto-completes

### ISSUE THREAD

**Issue by emibou** [luoliyan/chinese-support-redux#145]
I use this Add-On everyday (on Mac OS Catalina 10.15.5) and find it incredibly useful - thank you for your work!

Unfortunately, since I updated Anki to the latest version (2.1.29) a few days ago, the "Meaning" field no longer auto-completes when I create a new card. The other fields (Reading, Ruby, Color) seem to auto-complete fine as before.

Many thanks in advance for your help!

**Comment by 3ter** [comment]
[Anki 2.1.30](https://apps.ankiweb.net/) has just been released. I just checked and the auto-completion seems to work fine on a fresh install.

Does upgrading Anki (again) resolve your issue?

**Comment by emibou** [comment]
Thanks 3ter for the tip. The issue persisted even after upgrading to Anki 2.1.30, but I have just been able to resolve it by renaming the field on my cards from "Meaning" to "English" and now it works fine.

The Add-On page on ankiweb.net still mentions "Meaning" as one of the valid field names, but when checking the page config.json as recommended by that page, I realised the field "Meaning" is no longer listed but there is a field "English" which seems to have the same function.
